src/routes/google-routes.js

```diff
@@ -9,17 +9,18 @@ router.get(
   passportGoogle.authenticate('google', {
     session: false,
     scope: ['profile', 'email'],
-  })
-
+  }),
 );
 
 // callback route for google
 router.get(
   '/redirect',
```

Comment:
why not use '/callback' like you did for facebook

Reply:
The route is hosted for google already and it works. Had to change the one for facebook because it kept failing (I deleted the original hosted app and had to start over)